ansible/inventory/hosts

```diff
@@ -2,7 +2,7 @@
 
 [kstest-master]
 
-[kstest:all]
+[all:vars]
```

Comment:
I think this could be reverted from the commit before instead of having new commit.